### PR TITLE
Fix nxp 26026 publish fix

### PR DESCRIPTION
**Pull request by guirenard**

### CODE REVIEW

File: elements/nuxeo-publication/nuxeo-internal-publish.html [hunk 1]
```diff
@@ -73,9 +80,13 @@
         placeholder="[[i18n('publication.internal.location.placeholder')]]"
         selected-item="{{publishSpace}}"
         min-chars="0"
+        enrichers="permissions"
         page-provider="publish_space_suggestion"
         repository="[[document.repository]]">
       </nuxeo-document-suggestion>
+      <template is="dom-if" if="[[errorMessage]]">
+        <span class="horizontal layout error">&#8203;[[errorMessage]]</span>
```
Comment (Gabez0r) [inline elements/nuxeo-publication/nuxeo-internal-publish.html]: What's the purpose of `&#8203;` here?

Comment (guirenard) [reply elements/nuxeo-publication/nuxeo-internal-publish.html]: Inspired by https://github.com/nuxeo/nuxeo-web-ui/blob/master/elements/document/nuxeo-document-create.html#L176 I believe it is to highlight such error message:
error

Comment (Gabez0r) [reply elements/nuxeo-publication/nuxeo-internal-publish.html]: In that case, the highlight is done by
```
.error {
  border-left: 4px solid var(--nuxeo-warn-text);
  color: var(--nuxeo-text-default);
  padding-left: 8px;
}
```
I'm not sure anymore why the `&#8203;` is there, though.

**Comment by Gabez0r** [comment]
👍

File: elements/nuxeo-publication/nuxeo-internal-publish.html [hunk 2]
```diff
@@ -187,7 +198,16 @@
       },
 
       _canPublish: function() {
```
Comment (nmpcunha) [inline elements/nuxeo-publication/nuxeo-internal-publish.html]: Suggestion for readability:

`_canPublish: function() {
    this.errorMessage = null;
    if(!this.publishSpace) {
        return false;
    } 
    const hasPermission = this.hasPermission(this.publishSpace, 'AddChildren');
    if(!hasPermission) {
        this.errorMessage = this.i18n('publication.internal.location.error.noPermission');
    }
    return hasPermission;
},`

**Comment by nmpcunha** [comment]
👍

**Pull request merged**